ui-src/src/keyboard/actions.js

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Should we put `SET_G_KEYDOWN` and `UNSET_G_KEYDOWN` here too?

Reply:
Yeah, perhaps good to consolidate...

Reply:
right now it's inside goalCreation in the redux state (since it has to do with creating a goal). But I think it should be in keyboard, otherwise we'd put `SET_SHIFT_KEYDOWN` in under selection in the state

Reply:
Right... I should have thought to put it in `selection` feature

Reply:
Alternatively

Reply:
I did that consolidation into `keyboard` feature